### DISCUSSION TITLE
Art Class v2 - GeForce NOW

### DISCUSSION THREAD

**Discussion by giscus** [proudparrot2/artclass-v2#5] [Apps Coments]
# Art Class v2 - GeForce NOW

https://artclass.site/apps/geforce-now.html

**Comment by troycla** [comment]
pls fix this

**Comment by proudparrot2** [comment]
I will try my best - for now, you could try using https://moon.artclass.site (proxy) and type it in there

**Comment by proudparrot2** [comment]
Or https://lonely-bear-bonnet.cyclic.app/
